domain/mvc/router.go

```diff
@@ -69,6 +69,9 @@ type RouterUsecase interface {
 	// GetCustomDirectQuoteMultiPool calculates direct custom quote for given tokenIn and tokenOutDenom over given poolID route.
 	// Otherwise it implements same rules as GetCustomDirectQuote.
 	GetCustomDirectQuoteMultiPool(ctx context.Context, tokenIn sdk.Coin, tokenOutDenom []string, poolIDs []uint64) (domain.Quote, error)
+	// GetCustomDirectQuoteMultiPool calculates direct custom quote for given tokenOut and tokenInDenom over given poolID route.
+	// Otherwise it implements same rules as GetCustomDirectQuote.
```

Comment:
Let's be more specific about what ie means to implement the same rules

Reply:
I have updated docs to use different terminology to avoid duplicating `GetCustomDirectQuote` description for each method that uses it as underlying mechanism. Does it make sense?